index.html

```diff
@@ -122,7 +124,7 @@
                     ></span>
                   </div>
                 </template>
-                <div class="guesses-tile emoji-tile">
+                <div class="guesses-tile emoji-tile basis-1/6">
```

Comment:
this basically just sets width to 16%... which maybe works well enough now that we're using CSS grid etc